tasmota/support_features.ino

```diff
@@ -254,8 +254,8 @@ void GetFeatures(void)
 #ifdef USE_SHT
   feature_sns1 |= 0x00000100;  // xsns_07_sht1x.ino
 #endif
-#ifdef USE_HTU
-  feature_sns1 |= 0x00000200;  // xsns_08_htu21.ino
+#if defined(USE_HTU) || defined(USE_HDC1080)
+  feature_sns1 |= 0x00000200;  // xsns_08_htu21.ino or xsns_92_hdc1080.ino
 #endif
```

Comment:
Don't do this. If both HTU and HDC are connected use the correct online commands to disable one of them (your code must support this. See HTU code what I mean).

Don't update support_features.ino any way. I'll dot the administration.